core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -334,6 +334,17 @@ class SparkHadoopUtil extends Logging {
    * Stop the thread that does the delegation token updates.
    */
   private[spark] def stopExecutorDelegationTokenRenewer() {}
+
+  /**
+   * Disable the hadoop fs cache mechanism, otherwise DFSClient will use old token to connect nn.
+   */
+  private[spark]
+  def getConfBypassingFSCache(hadoopConf: Configuration, path: Path): Configuration = {
```

Comment:
I keep this function name since this function is not a general methon, it only fresh the `cache` configuration.